tests/integration_tests/test_kani.py

```diff
@@ -24,10 +29,8 @@ def test_kani(results_dir):
     # -j enables kani harnesses to be verified in parallel (required to keep CI time low)
     # --output-format terse is required by -j
     # --enable-unstable is needed for each of the above
-    rc, stdout, stderr = utils.run_cmd(
+    _, stdout, _ = utils.run_cmd(
         "cargo kani --enable-unstable --enable-stubbing --restrict-vtable -j --output-format terse"
     )
 
-    assert rc == 0, stderr
-
```

Comment:
Why remove that?

Reply:
It is already checked by `run_cmd`: 
https://github.com/firecracker-microvm/firecracker/blob/532677328480b7f4149192ec121ac076451ac098/tests/framework/utils.py#L482-L487

Reply:
oh awesome. Thanks.